### PR TITLE
Add Beta & Blog parts to feature template

### PR DESCRIPTION
**Pull request by mbroz2**
Update feature template with beta and blog post specific parts.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -69,6 +79,9 @@ All features (both "Design Approved" and "No Design Approved")
 - [ ] Ship Readiness Review and Release Notes completed (Epic owner / Feature owner / Release PM)
 - [ ] Github Epic and Epic's issues are closed / complete. All PRs are committed to the master branch. (Epic owner / Feature owner / Backlog Subtribe PM)
 
+1 week before GA
+- [ ] Create, populate, and link to the [Blog post issue](https://github.com/OpenLiberty/open-liberty/issues/new?assignees=lauracowen%2C+jakub-pomykala&labels=&template=blog_post_ga_release.md&title=GA+BLOG+-+title_of_your_update)
```
Comment (tevans78) [inline .github/ISSUE_TEMPLATE/feature.md:83]: Do you really want Laura and Jakub hardcoded into this query?

**Comment by tevans78** [comment]
> Read and understand this completely, then delete this entire template. If a reviewer or merger sees this template, they should fail the review or merge.
@mbroz2 So should I fail the review for this template update?!

File: .github/ISSUE_TEMPLATE/feature.md (hunk 1 above)
Comment (lauracowen) [reply .github/ISSUE_TEMPLATE/feature.md:83]: If they need to be assigned to anyone, someone needs to be hardcoded in I think or they have to guess at who to assign it to. But probably Jakub and Austin (https://github.com/austin0) would be better now? Or just leave it blank and they'll work from Sam's query - feels a bit scary without assignees though - could just get lost in the ether....

**Review by lauracowen — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/feature.md (hunk 1 above)
Comment (tevans78) [reply .github/ISSUE_TEMPLATE/feature.md:83]: from a developer's perspective, isn't the label enough? it is not up to them to ensure that the issue is assigned? and what happens if the team members change?

Comment (lauracowen) [reply .github/ISSUE_TEMPLATE/feature.md:83]: I completely agree. I just see them tend to assign the issues to someone they think they should go to. But maybe that doesn't matter. If they do assign it, fine. If they don't, also fine.

Comment (mbroz2) [reply .github/ISSUE_TEMPLATE/feature.md:83]: I don't mind one way or the other, but those links are the ones that are used by the blog post templates: https://github.com/OpenLiberty/open-liberty/issues/new/choose
which is why I used them.  If we don't want to have default assignees on the blog posts issues, then the templates should be updated as well.

Comment (mbroz2) [reply .github/ISSUE_TEMPLATE/feature.md:83]: Given the above, I'm going to merge this as-is, and if we want to remove default assignees from the blog post, we can do it in all 3 templates in a subsequent PR, but I'll leave that call for @lauracowen to make

**Pull request merged**